src_py/sysfont.py

```diff
@@ -50,6 +50,28 @@ def _addfont(name, bold, italic, font, fontdict):
     fontdict[name][bold, italic] = font
 
 
+def _initsysfontsusers_win32(fonts):
+    """initialize fonts dictionary on Windows from all users
+
+    Uses path: "C:\\Users\\<user>\\AppData\\Local\\Microsoft\\Windows\\Fonts".
+
+    Parameters:
+        fonts: The pygame font dictionary
+
+    """
+    user_path = os.environ["USERPROFILE"]
+    try:
+        fonts_path = f"{user_path}\\AppData\\Local\\Microsoft\\Windows\\Fonts"
+        for font in os.listdir(fonts_path):
+            font = font.lower()
+            if font.endswith(".ttf") or font.endswith(".otf"):
```

Comment:
There's a frozenset "Opentype_extensions" you should reference instead of doing your tests like this.